content/docs/command-reference/status.md

```diff
@@ -182,14 +182,14 @@ This shows that for stage `bar.dvc`, the dependency `foo` and the
 ```dvc
 $ dvc status foo.dvc dobar
 foo.dvc:
```

Comment:
Double check please

Reply:
It's an example for specific files/dirs. So our first target here is a `.dvc` file and second one is a stage. Output is also displayed in same order.

Reply:
OK. Can we be consistent with the sample .dvc file or stage naming in all examples? E.g. in the first one use `dofoo` or/and `dobar`. Thanks

Reply:
Changed.